src/CarbonAware.WebApi/src/Models/EmissionsForecastBatchDTO.cs

```diff
@@ -6,9 +6,9 @@ namespace CarbonAware.WebApi.Models;
 [Serializable]
 public record EmissionsForecastBatchDTO : EmissionsForecastBaseDTO
 {
-  /// <summary>The historical time used to fetch the most recent forecast as of that time.</summary>
-  /// <example>2022-06-01T00:03:30Z</example>
-  [JsonPropertyName("requestedAt")]
-  [Required()]
-  public DateTimeOffset RequestedAt { get; set; }
+    /// <summary>The historical time used to fetch the most recent forecast as of that time.</summary>
```

Comment:
NOTE that this definition is changed in https://github.com/microsoft/carbon-aware-sdk/pull/109/files#diff-635df06bb41839059ce95c773006ff8a174a43134b08222a14ada6c294bee53f 

Whoever merges first may need to update accordingly